AutomationFramework/src/test/java/steam/tests/ValidateCategoriesTest.java

```diff
@@ -0,0 +1,21 @@
+package steam.tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import steam.pages.MainPage;
+import steam.steps.MainPageSteps;
+
+public class ValidateCategoriesTest extends BaseTest{
+    private MainPageSteps mainPageSteps;
+    @Test
+    public void validateCategoriesTest() {
+        MainPage mainPage = new MainPage();
+        mainPage.navigateToCategories();
+        mainPageSteps = new MainPageSteps();
+        mainPageSteps.waitingOfDisplayedListsOfCategories();
+        Assert.assertTrue(mainPageSteps.equalsListOfCategories("action"));
+        Assert.assertTrue(mainPageSteps.equalsListOfCategories("rpg"));
+        Assert.assertTrue(mainPageSteps.equalsListOfCategories("strategy"));
```

Comment:
you have 3 identical calls of method `mainPageSteps.equalsListOfCategories`
You should make method inside mainPage that returns list of categories, than you can assert that your actual list of categories contains elements from expected list